junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestExtensionTests.java

```diff
@@ -69,8 +69,8 @@ void streamsReturnedByProvidersAreClosedWhenCallingProvide() {
 		Stream<TestTemplateInvocationContext> stream = this.parameterizedTestExtension.provideTestTemplateInvocationContexts(
 			extensionContext);
 
-		//cause the stream to be evaluated
-		stream.count();
+		// cause the stream to be evaluated
+		long unused = stream.count();
```

Comment:
I definitely don't think the rule is worth keeping enabled for tests, since we often invoke a method that will throw an exception and don't care about the return value (that will never exist).